examples/pytorch/gat/main.py

```diff
@@ -1,19 +1,32 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 
+from typing import Optional, Dict
+import os
+import platform
```

Comment:
import is not used

Reply:
Wrong PR, these examples will be overwritten by 2/3 per previous discussion.

Reply:
Please submit atomic PRs. If these lines are not touched in [2/3] I can’t discover these issues there.

Reply:
Okay merged